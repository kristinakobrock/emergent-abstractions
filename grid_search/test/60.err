  0%|                                                                                                     | 0/50 [00:00<?, ?it/s]  4%|###7                                                                                         | 2/50 [00:00<00:02, 18.10it/s] 10%|#########3                                                                                   | 5/50 [00:00<00:02, 22.29it/s] 16%|##############8                                                                              | 8/50 [00:00<00:01, 23.73it/s] 22%|####################2                                                                       | 11/50 [00:00<00:01, 25.79it/s] 28%|#########################7                                                                  | 14/50 [00:00<00:01, 25.65it/s] 34%|###############################2                                                            | 17/50 [00:00<00:01, 24.78it/s] 42%|######################################6                                                     | 21/50 [00:00<00:01, 26.44it/s] 48%|############################################1                                               | 24/50 [00:00<00:01, 24.95it/s] 54%|#################################################6                                          | 27/50 [00:01<00:00, 23.93it/s] 60%|#######################################################2                                    | 30/50 [00:01<00:00, 22.33it/s] 66%|############################################################7                               | 33/50 [00:01<00:00, 23.95it/s] 72%|##################################################################2                         | 36/50 [00:01<00:00, 24.25it/s] 80%|#########################################################################6                  | 40/50 [00:01<00:00, 27.18it/s] 86%|###############################################################################1            | 43/50 [00:01<00:00, 26.80it/s] 92%|####################################################################################6       | 46/50 [00:01<00:00, 27.64it/s] 98%|##########################################################################################1 | 49/50 [00:01<00:00, 25.88it/s]100%|############################################################################################| 50/50 [00:02<00:00, 24.97it/s]
  0%|                                                                                                     | 0/13 [00:00<?, ?it/s] 23%|#####################4                                                                       | 3/13 [00:00<00:00, 21.58it/s] 54%|##################################################                                           | 7/13 [00:00<00:00, 26.26it/s] 77%|######################################################################7                     | 10/13 [00:00<00:00, 26.00it/s]100%|############################################################################################| 13/13 [00:00<00:00, 28.32it/s]
