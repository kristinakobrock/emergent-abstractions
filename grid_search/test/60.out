# ["--attributes=5", "--values=4", "--game_size=10", "--vocab_size_factor=3", "--n_epochs=60", "--batch_size=64", "--learning_rate=0.001", "--hidden_size=128", "--temp_update=0.97", "--temperature=2"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.6768527030944824, "acc": 0.61033695936203, "length": 3.48710560798645, "mode": "train", "epoch": 1}
{"loss": 0.6683945059776306, "acc": 0.624804675579071, "length": 1.0, "mode": "test", "epoch": 1}
{"loss": 0.6635691523551941, "acc": 0.629021167755127, "length": 3.313365936279297, "mode": "train", "epoch": 2}
{"loss": 0.6717745065689087, "acc": 0.623046875, "length": 1.0, "mode": "test", "epoch": 2}
{"loss": 0.6595584154129028, "acc": 0.6292084455490112, "length": 3.0838873386383057, "mode": "train", "epoch": 3}
{"loss": 0.6546401381492615, "acc": 0.6277893781661987, "length": 1.96875, "mode": "test", "epoch": 3}
{"loss": 0.6516538858413696, "acc": 0.6295472383499146, "length": 3.3220503330230713, "mode": "train", "epoch": 4}
{"loss": 0.6352814435958862, "acc": 0.6431640982627869, "length": 4.0, "mode": "test", "epoch": 4}
{"loss": 0.6279476881027222, "acc": 0.6449331045150757, "length": 3.730616807937622, "mode": "train", "epoch": 5}
{"loss": 0.6279467940330505, "acc": 0.6507812142372131, "length": 4.0, "mode": "test", "epoch": 5}
{"loss": 0.6054348945617676, "acc": 0.6755428314208984, "length": 3.7763524055480957, "mode": "train", "epoch": 6}
{"loss": 0.6092823147773743, "acc": 0.6775389909744263, "length": 4.0, "mode": "test", "epoch": 6}
{"loss": 0.5755845904350281, "acc": 0.7079803943634033, "length": 3.8795783519744873, "mode": "train", "epoch": 7}
{"loss": 0.6044121980667114, "acc": 0.6949219107627869, "length": 4.0, "mode": "test", "epoch": 7}
{"loss": 0.5432494878768921, "acc": 0.7272693514823914, "length": 3.958677053451538, "mode": "train", "epoch": 8}
{"loss": 0.5819534659385681, "acc": 0.7126952409744263, "length": 4.0, "mode": "test", "epoch": 8}
{"loss": 0.5256254076957703, "acc": 0.7398644089698792, "length": 3.9751405715942383, "mode": "train", "epoch": 9}
{"loss": 0.5932357311248779, "acc": 0.6996093392372131, "length": 4.0, "mode": "test", "epoch": 9}
{"loss": 0.512424111366272, "acc": 0.7475697994232178, "length": 3.984004497528076, "mode": "train", "epoch": 10}
{"loss": 0.574364423751831, "acc": 0.7031249403953552, "length": 4.0, "mode": "test", "epoch": 10}
{"loss": 0.5028475522994995, "acc": 0.7552448511123657, "length": 3.9904911518096924, "mode": "train", "epoch": 11}
{"loss": 0.5549051761627197, "acc": 0.7216796875, "length": 4.0, "mode": "test", "epoch": 11}
{"loss": 0.49389252066612244, "acc": 0.7650508880615234, "length": 3.991279125213623, "mode": "train", "epoch": 12}
{"loss": 0.543737530708313, "acc": 0.7230468392372131, "length": 4.0, "mode": "test", "epoch": 12}
{"loss": 0.48290470242500305, "acc": 0.7699450850486755, "length": 3.9900431632995605, "mode": "train", "epoch": 13}
{"loss": 0.5599640607833862, "acc": 0.7431640625, "length": 4.0, "mode": "test", "epoch": 13}
{"loss": 0.4763958752155304, "acc": 0.7784402370452881, "length": 3.988416910171509, "mode": "train", "epoch": 14}
{"loss": 0.5409676432609558, "acc": 0.7486328482627869, "length": 4.0, "mode": "test", "epoch": 14}
{"loss": 0.4691525995731354, "acc": 0.7795366048812866, "length": 3.983657121658325, "mode": "train", "epoch": 15}
{"loss": 0.5176467299461365, "acc": 0.7681641578674316, "length": 4.0, "mode": "test", "epoch": 15}
{"loss": 0.4579785168170929, "acc": 0.7912931442260742, "length": 3.986299991607666, "mode": "train", "epoch": 16}
{"loss": 0.49674510955810547, "acc": 0.7763671278953552, "length": 4.0, "mode": "test", "epoch": 16}
{"loss": 0.4446609616279602, "acc": 0.7991486191749573, "length": 3.989056348800659, "mode": "train", "epoch": 17}
{"loss": 0.48648130893707275, "acc": 0.7699218392372131, "length": 4.0, "mode": "test", "epoch": 17}
{"loss": 0.4414711892604828, "acc": 0.8012564182281494, "length": 3.9912755489349365, "mode": "train", "epoch": 18}
{"loss": 0.4791746437549591, "acc": 0.7734375, "length": 4.0, "mode": "test", "epoch": 18}
{"loss": 0.4343995153903961, "acc": 0.8058360815048218, "length": 3.992483615875244, "mode": "train", "epoch": 19}
{"loss": 0.4749855697154999, "acc": 0.7800780534744263, "length": 4.0, "mode": "test", "epoch": 19}
{"loss": 0.42792677879333496, "acc": 0.8060107231140137, "length": 3.994912624359131, "mode": "train", "epoch": 20}
{"loss": 0.48075827956199646, "acc": 0.7818359136581421, "length": 4.0, "mode": "test", "epoch": 20}
{"loss": 0.42846545577049255, "acc": 0.8095945119857788, "length": 3.995189666748047, "mode": "train", "epoch": 21}
{"loss": 0.46114516258239746, "acc": 0.7859374284744263, "length": 4.0, "mode": "test", "epoch": 21}
{"loss": 0.42013105750083923, "acc": 0.8124043345451355, "length": 3.9921231269836426, "mode": "train", "epoch": 22}
{"loss": 0.4676276743412018, "acc": 0.7851561903953552, "length": 4.0, "mode": "test", "epoch": 22}
{"loss": 0.41475579142570496, "acc": 0.8134321570396423, "length": 3.9959723949432373, "mode": "train", "epoch": 23}
{"loss": 0.4674573242664337, "acc": 0.7837890982627869, "length": 4.0, "mode": "test", "epoch": 23}
{"loss": 0.4131510555744171, "acc": 0.8166773915290833, "length": 3.9959535598754883, "mode": "train", "epoch": 24}
{"loss": 0.46575242280960083, "acc": 0.7921875715255737, "length": 4.0, "mode": "test", "epoch": 24}
{"loss": 0.4070883095264435, "acc": 0.8173630237579346, "length": 3.996872901916504, "mode": "train", "epoch": 25}
{"loss": 0.47131168842315674, "acc": 0.7818359732627869, "length": 4.0, "mode": "test", "epoch": 25}
{"loss": 0.40409156680107117, "acc": 0.8196452260017395, "length": 3.997502565383911, "mode": "train", "epoch": 26}
{"loss": 0.465313196182251, "acc": 0.7896484732627869, "length": 4.0, "mode": "test", "epoch": 26}
{"loss": 0.3970027565956116, "acc": 0.8233464956283569, "length": 3.997231960296631, "mode": "train", "epoch": 27}
{"loss": 0.46576952934265137, "acc": 0.7843749523162842, "length": 4.0, "mode": "test", "epoch": 27}
{"loss": 0.3923985958099365, "acc": 0.8265932202339172, "length": 3.997421979904175, "mode": "train", "epoch": 28}
{"loss": 0.4700081944465637, "acc": 0.7939453125, "length": 4.0, "mode": "test", "epoch": 28}
{"loss": 0.38448789715766907, "acc": 0.8305361866950989, "length": 3.997925281524658, "mode": "train", "epoch": 29}
{"loss": 0.458185613155365, "acc": 0.7941405773162842, "length": 4.0, "mode": "test", "epoch": 29}
{"loss": 0.3773910105228424, "acc": 0.8358990550041199, "length": 3.9974021911621094, "mode": "train", "epoch": 30}
{"loss": 0.4516473412513733, "acc": 0.8001952767372131, "length": 4.0, "mode": "test", "epoch": 30}
{"loss": 0.37337955832481384, "acc": 0.8376717567443848, "length": 3.9991843700408936, "mode": "train", "epoch": 31}
{"loss": 0.4357537031173706, "acc": 0.8064451813697815, "length": 4.0, "mode": "test", "epoch": 31}
{"loss": 0.3675592243671417, "acc": 0.8416850566864014, "length": 3.9978854656219482, "mode": "train", "epoch": 32}
{"loss": 0.42912358045578003, "acc": 0.8109374642372131, "length": 4.0, "mode": "test", "epoch": 32}
{"loss": 0.3664657771587372, "acc": 0.8454895615577698, "length": 3.9995102882385254, "mode": "train", "epoch": 33}
{"loss": 0.4192051291465759, "acc": 0.826171875, "length": 4.0, "mode": "test", "epoch": 33}
{"loss": 0.3570546805858612, "acc": 0.8525521755218506, "length": 3.999002456665039, "mode": "train", "epoch": 34}
{"loss": 0.4167863726615906, "acc": 0.8275390267372131, "length": 4.0, "mode": "test", "epoch": 34}
{"loss": 0.36007794737815857, "acc": 0.8523139953613281, "length": 3.9983015060424805, "mode": "train", "epoch": 35}
{"loss": 0.426294207572937, "acc": 0.8246093392372131, "length": 4.0, "mode": "test", "epoch": 35}
{"loss": 0.34882020950317383, "acc": 0.8573130369186401, "length": 3.997208595275879, "mode": "train", "epoch": 36}
{"loss": 0.4111367464065552, "acc": 0.8304687142372131, "length": 4.0, "mode": "test", "epoch": 36}
{"loss": 0.34786146879196167, "acc": 0.855870246887207, "length": 3.9998700618743896, "mode": "train", "epoch": 37}
{"loss": 0.4022470712661743, "acc": 0.8359375, "length": 4.0, "mode": "test", "epoch": 37}
{"loss": 0.3413897156715393, "acc": 0.8615111708641052, "length": 3.998065233230591, "mode": "train", "epoch": 38}
{"loss": 0.39705970883369446, "acc": 0.8367186784744263, "length": 4.0, "mode": "test", "epoch": 38}
{"loss": 0.34006360173225403, "acc": 0.8610004782676697, "length": 3.9997880458831787, "mode": "train", "epoch": 39}
{"loss": 0.41219472885131836, "acc": 0.8318358659744263, "length": 4.0, "mode": "test", "epoch": 39}
{"loss": 0.33521419763565063, "acc": 0.8626888990402222, "length": 3.9991278648376465, "mode": "train", "epoch": 40}
{"loss": 0.4032326936721802, "acc": 0.8349609375, "length": 4.0, "mode": "test", "epoch": 40}
{"loss": 0.33493420481681824, "acc": 0.8659979701042175, "length": 3.9986934661865234, "mode": "train", "epoch": 41}
{"loss": 0.4058660864830017, "acc": 0.8339843153953552, "length": 4.0, "mode": "test", "epoch": 41}
{"loss": 0.33114001154899597, "acc": 0.8676044940948486, "length": 3.998944044113159, "mode": "train", "epoch": 42}
{"loss": 0.3924349546432495, "acc": 0.8402343392372131, "length": 4.0, "mode": "test", "epoch": 42}
{"loss": 0.3384656608104706, "acc": 0.8606563210487366, "length": 3.999830484390259, "mode": "train", "epoch": 43}
{"loss": 0.3844073414802551, "acc": 0.8445311784744263, "length": 4.0, "mode": "test", "epoch": 43}
{"loss": 0.3280695974826813, "acc": 0.8656752109527588, "length": 3.99985671043396, "mode": "train", "epoch": 44}
{"loss": 0.39250653982162476, "acc": 0.8414062857627869, "length": 4.0, "mode": "test", "epoch": 44}
{"loss": 0.3343581259250641, "acc": 0.8631690740585327, "length": 3.999950647354126, "mode": "train", "epoch": 45}
{"loss": 0.40290367603302, "acc": 0.8351563215255737, "length": 4.0, "mode": "test", "epoch": 45}
{"loss": 0.3261747360229492, "acc": 0.865655779838562, "length": 3.9994828701019287, "mode": "train", "epoch": 46}
{"loss": 0.41529905796051025, "acc": 0.8298827409744263, "length": 4.0, "mode": "test", "epoch": 46}
{"loss": 0.3321034610271454, "acc": 0.8651048541069031, "length": 3.9998793601989746, "mode": "train", "epoch": 47}
{"loss": 0.3981640338897705, "acc": 0.8363280892372131, "length": 4.0, "mode": "test", "epoch": 47}
{"loss": 0.32347115874290466, "acc": 0.8673864603042603, "length": 3.9998903274536133, "mode": "train", "epoch": 48}
{"loss": 0.38496696949005127, "acc": 0.8445311784744263, "length": 4.0, "mode": "test", "epoch": 48}
{"loss": 0.3179834485054016, "acc": 0.8700032830238342, "length": 3.99955677986145, "mode": "train", "epoch": 49}
{"loss": 0.37679576873779297, "acc": 0.8499999642372131, "length": 4.0, "mode": "test", "epoch": 49}
{"loss": 0.31077519059181213, "acc": 0.8771926164627075, "length": 3.9998958110809326, "mode": "train", "epoch": 50}
{"loss": 0.3668438494205475, "acc": 0.8552733659744263, "length": 4.0, "mode": "test", "epoch": 50}
{"loss": 0.3015458881855011, "acc": 0.8784491419792175, "length": 3.9999961853027344, "mode": "train", "epoch": 51}
{"loss": 0.3744423985481262, "acc": 0.8466796875, "length": 4.0, "mode": "test", "epoch": 51}
{"loss": 0.2991523742675781, "acc": 0.8802043199539185, "length": 3.9996442794799805, "mode": "train", "epoch": 52}
{"loss": 0.3696841597557068, "acc": 0.8562498688697815, "length": 4.0, "mode": "test", "epoch": 52}
{"loss": 0.2939058840274811, "acc": 0.882940948009491, "length": 3.998041868209839, "mode": "train", "epoch": 53}
{"loss": 0.35567331314086914, "acc": 0.8552733659744263, "length": 4.0, "mode": "test", "epoch": 53}
{"loss": 0.291490375995636, "acc": 0.8840827345848083, "length": 3.9997458457946777, "mode": "train", "epoch": 54}
{"loss": 0.3620874881744385, "acc": 0.8619140982627869, "length": 4.0, "mode": "test", "epoch": 54}
{"loss": 0.2952878773212433, "acc": 0.8833519220352173, "length": 3.999990463256836, "mode": "train", "epoch": 55}
{"loss": 0.3539663553237915, "acc": 0.8591796159744263, "length": 4.0, "mode": "test", "epoch": 55}
{"loss": 0.2946913540363312, "acc": 0.881815493106842, "length": 3.997642755508423, "mode": "train", "epoch": 56}
{"loss": 0.361162930727005, "acc": 0.8648437857627869, "length": 4.0, "mode": "test", "epoch": 56}
{"loss": 0.28592029213905334, "acc": 0.8905850052833557, "length": 3.9998278617858887, "mode": "train", "epoch": 57}
{"loss": 0.36035963892936707, "acc": 0.8544921278953552, "length": 4.0, "mode": "test", "epoch": 57}
{"loss": 0.28440943360328674, "acc": 0.8892647624015808, "length": 3.999506950378418, "mode": "train", "epoch": 58}
{"loss": 0.3412896394729614, "acc": 0.8675780892372131, "length": 4.0, "mode": "test", "epoch": 58}
{"loss": 0.27286556363105774, "acc": 0.8939557671546936, "length": 3.999976634979248, "mode": "train", "epoch": 59}
{"loss": 0.3357483148574829, "acc": 0.8744140267372131, "length": 4.0, "mode": "test", "epoch": 59}
{"loss": 0.2747827470302582, "acc": 0.8936712145805359, "length": 3.9999847412109375, "mode": "train", "epoch": 60}
{"loss": 0.33193498849868774, "acc": 0.8759765028953552, "length": 4.0, "mode": "test", "epoch": 60}
test accuracy: 0.7749999761581421
